framework/infrastructure/gcp/compute.py

```diff
@@ -39,6 +39,7 @@ class ComputeV1(
     _WAIT_FOR_BACKEND_SLEEP_SEC = 4
     _WAIT_FOR_OPERATION_SEC = 60 * 10
     gfe_debug_header: Optional[str]
+    add_dualstack_support: Optional[bool]
```

Comment:
nit: no need `Optional` for booleans, we can simply set it to False
```suggestion
    add_dualstack_support: bool = False
```

Reply:
Done